backbone/build.gradle

```diff
@@ -33,7 +33,7 @@ android {
         exclude 'META-INF/NOTICE.txt'
     }
 
-    resourcePrefix 'rsb_'
+//    resourcePrefix 'rsb_'
```

Comment:
I'm not sure of the root cause of why this is throwing so many warnings.  But, it is actually something we should in the gradle file, maybe @liujoshua knows why its not working correctly?